api/v1/helpers/verbs/utils.js

```diff
@@ -567,6 +567,8 @@ function cleanAndStripNulls(obj) {
       // to pass swagger's schema validation
       if (key === 'parentAbsolutePath' && !o[key]) {
         o[key] = '';
+      } else if (key === 'parentId' && !o[key]) {
+        o[key] = null;
```

Comment:
previously on PUT with isPublished and name fields only, the parentId field was not being returned. This allows parentId to be returned even though it has a null value